packages/melody-extension-core/src/parser/mount.js

```diff
@@ -50,7 +61,58 @@ export const MountParser = {
             argument = parser.matchExpression();
         }
 
-        const mountStatement = new MountStatement(name, source, key, argument);
+        if (async) {
+            if (tokens.nextIf(Types.SYMBOL, 'delay')) {
+                tokens.expect(Types.SYMBOL, 'placeholder');
```

Comment:
Does the "placeholder" token actually have any functionality? If not, I would consider dropping it, and also the "by" (or making it optional), so that only "delay" is left.

Reply:
None of them have any functional meaning apart from hopefully explaining the concept better. I actually started with `delay by` but when showing it to engineers they were wondering why we'd want to delay loading the async component.

The more explicit `delay placeholder by` seemed easier to understand and easier on the eye.